planning/surround_obstacle_checker/src/node.cpp

```diff
@@ -205,13 +281,13 @@ void SurroundObstacleCheckerNode::onTimer()
     debug_ptr_->publishFootprints();
   }
 
-  if (node_param_.use_pointcloud && !pointcloud_ptr_) {
+  if (node_param_.pointcloud_enable_check && !pointcloud_ptr_) {
     RCLCPP_INFO_THROTTLE(
       this->get_logger(), *this->get_clock(), 5000 /* ms */, "waiting for pointcloud info...");
     return;
   }
 
-  if (node_param_.use_dynamic_object && !object_ptr_) {
+  if (!object_ptr_) {
```

Comment:
@takayuki5168 Is it ok to remove `use_dynamic_object` because there are some products which doesn't use perception module?